content/en/docs/architecture/ci-operator.md

```diff
@@ -243,7 +243,7 @@ promotion:
   tag: "4.5"
 {{< / highlight >}}
 
-## Describing Inclusion in an OpenShift Release
+## Describing OpenShift Releases Involved in Tests {#describing-inclusion-in-an-openshift-release}
```

Comment:
I've changed the heading because I've met at least one person who skipped the section after thinking it's not relevant for them (because their component is not included in openshift release).

I've kept the previous anchor to not break existing links.